### PR TITLE
Remove y-up-to-z-up transform for batched tiles

### PR DESCRIPTION
**Pull request by pmconne**
As a result of [changes to exporter](https://dev.azure.com/bentleycs/iModelTechnologies/_git/graphics-exporter-services/pullrequest/321606).

### CODE REVIEW

**Comment by pmconne** [comment]
@wgoehrig FYI you will need changes to exporter if you consume new frontend-tiles package containing this change.

File: common/changes/@itwin/frontend-tiles/pmc-batched-tile-feature-table_2023-03-15-19-00.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/frontend-tiles",
+      "comment": "Remove y-up-to-z-up display transform.",
```
Comment (aruniverse) [inline common/changes/@itwin/frontend-tiles/pmc-batched-tile-feature-table_2023-03-15-19-00.json:5]:
```suggestion
      "comment": "",
```

**Review by aruniverse — APPROVED** [review]

**Review by danieliborra — APPROVED** [review]

File: common/changes/@itwin/frontend-tiles/pmc-batched-tile-feature-table_2023-03-15-19-00.json (hunk 1 above)
Comment (pmconne) [reply common/changes/@itwin/frontend-tiles/pmc-batched-tile-feature-table_2023-03-15-19-00.json:5]: It's an actual functional change to the package, not an internal thing (to the extent that this entire package isn't "internal").

Comment (aruniverse) [reply common/changes/@itwin/frontend-tiles/pmc-batched-tile-feature-table_2023-03-15-19-00.json:5]: true; but this pkg has never been published (officially as a stable version (non-prelease)) so dont think its needed, but will resolve

**Pull request merged**